src/clj/game/cards/identities.clj

```diff
@@ -29,7 +29,46 @@
 
 ;;; Card definitions
 (def cards-identities
-  {"Adam: Compulsive Hacker"
+  {
+   "419: Amoral Scammer"
+   {:events {:corp-install
+             {:delayed-completion true
+              :req (req (first-event? state :corp :corp-install))
+              :once :per-turn
+              :effect
+              (req (show-wait-prompt state :corp "Runner to use 419: Amoral Scammer")
+                     (let [itarget target]
+                       (continue-ability
+                         state side
+                         {:optional
+                          {:prompt "Expose installed card unless Corp pays 1 [Credits]?"
+                           :player :runner
+                           :no-ability {:effect (req (clear-wait-prompt state :corp))}
+                           :yes-ability
+                           {:effect (req (clear-wait-prompt state :corp)
+                                         (if (not (can-pay? state :corp nil :credit 1))
+                                           (do
+                                             (toast state :corp "Cannot afford to pay 1 [Credits] to block card exposure" "info")
+                                             (expose state side itarget))
+                                           (do
+                                             (show-wait-prompt state :runner "Corp decision")
+                                             (continue-ability
+                                               state side
+                                               {:optional
+                                                {:prompt "Pay 1 [Credits] to prevent exposure of installed card?"
+                                                 :player :corp
+                                                 :no-ability
+                                                 {:effect (req (expose state side itarget)
```

Comment:
Same as before; this no-ability should be delayed completion and then pass eid to expose.